client/py/tests/test_io_reader.py

```diff
@@ -59,3 +62,62 @@ def test_read(self, valid_file: RowFileReader):
             assert len(d) == 1
             count += 1
         assert count == 4
+
+
+@pytest.mark.tdms
```

Comment:
We might eventually want to generalize this test suite to work with multiple column reader types (i.e. parameterize), but this LGTM for now.